{{cookiecutter.project_slug}}/include/{{cookiecutter.project_slug}}/example.hpp

```diff
@@ -1,6 +1,35 @@
 #pragma once
 
-#include <iostream>
+#include <filesystem>
+
+#include "emp/config/ArgManager.hpp"
+#include "emp/prefab/ConfigPanel.hpp"
+#include "emp/web/UrlParams.hpp"
+#include "emp/web/web.hpp"
+
+#include "{{cookiecutter.project_slug}}/ProjectConfig.hpp"
+
+void AddExistingConfig({{ cookiecutter.project_name.replace(' ', '') }}Config & config, emp::ArgManager & am) {
+  if(std::filesystem::exists("{{cookiecutter.project_slug}}.cfg")) {
+    std::cout << "Configuration read from {{cookiecutter.project_slug}}.cfg" << "\n";
+    config.Read("{{cookiecutter.project_slug}}.cfg");
+  }
+  am.UseCallbacks();
+  if (am.HasUnused())
+    std::exit(EXIT_FAILURE);
+}
+
+void SetupProjectConfig({{ cookiecutter.project_name.replace(' ', '') }}Config & config)  {
```

Comment:
rename so it's clear this is web entry point and one below is native entry point